lib/matplotlib/mlab.py

```diff
@@ -930,7 +930,7 @@ def scotts_factor(self):
 
     def silverman_factor(self):
         return np.power(
-            self.num_dp * (self.dim + 2.0) / 4.0, -1. / (self.dim + 4))
+            self.num_dp * (self.dim + 2) / 4, -1. / (self.dim + 4))
```

Comment:
```suggestion
            self.num_dp * (self.dim + 2) / 4, -1 / (self.dim + 4))
```

Reply:
Okay, so it is not enough to search for `\d\.0\b`. There is also `\d\.`. :-) Thank you.